src/App.js

```diff
@@ -91,8 +102,10 @@ export default function Game() {
 
   let moves = history.map((squares, move) => {
     let description;
+    let row = (Math.floor(indexes[move - 1] / 3)) +1;
+    let col = (indexes[move - 1] % 3) + 1;
     if (move > 0) {
-      description = 'Go to move #' + move;
+      description = 'Go to move #' + move + ` (${row}, ${col})`;
```

Comment:
Nice solution with math.floor and %.
Please add some default values because history displays NaN.

Reply:
Done.